tests/unit_tests/test_lakefs.py

```diff
@@ -0,0 +1,142 @@
+import unittest
+from typing import Any
+from unittest.mock import patch
+
+import pytest
+import requests_mock
+from requests_mock.mocker import Mocker
+
+from langchain_community.document_loaders.lakefs import LakeFSLoader
```

Comment:
The import from `langchain_community.document_loaders.lakefs` is not what I saw running in the sample code.
Based on the code it was from `langchain_lakefs.document_loaders`.